task_11/src/css/style.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think .map files is not required here.

Reply:
removed map file. But can you explain is there a need or some requirements that  it doesn't need to use or keep a map file?

Reply:
You need map files for development, or to be more specified, for debug. It _maps_ your source source and compiled code. Developer tools, like Chrome's one, pick it up and show you pointers  to selectors from original files instead of equal ones from compiled file.

But in most cases you do not need to push compiled files into the repository. And source maps as well.

Compiled files in the repository do mess there. It makes reviewing more complicated because reviewer need to check is file generated or not, scroll down big generated files and so on.

With not minimized SVG files (if they are not written by hand) the same situation.

So, we do not need generated files in the repository. Source maps are kind of it. When someone wants to debug the code, he can download, build and take a look on it.

Reply:
Thanks for such a detailed answer
Now it's finally clear how and when to use map files